kvstore/kvstore_server.cpp

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+#include <grpc/grpc.h>
+#include <unordered_map>
+
+class KvstoreServiceImpl final :  public KeyValueStore::Service {
```

Comment:
Move the definitions to the .h file and then you can have the functions alone here.

Reply:
Seperated like this?